GHzDACs/ghz_fpga_server.py

```diff
@@ -607,13 +612,15 @@ def run(self, runners, reps, setupPkts, setupState, sync, getTimingData,
         # setupState is a set
         setupPkts.extend(pkt for pkt, state in boardSetupPkts)
         setupState.update(state for pkt, state in boardSetupPkts)
-        
+
         try:
             yield self.pipeSemaphore.acquire()
+            logging.info('pipe semaphore acquired')
```

Comment:
nvm, I see you've set the default level to warning, so I guess these are off by default.